.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+---
+version: 2.1
+
+commands:
+  unittests:
+    steps:
+      
+      - checkout
+
+      - run:
+          name: install library
+          command: ./setup.py install --user
+
+      - run:
+          name: install tox
+          command: pip3 install tox
```

Comment:
We should lock the version to ensure it works after breaking changes in tox etc.